spec/features/user_creates_new_animal_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe AnimalsController, type: :controller do
+  describe "POST#create" do
+    it "creates a new Animal" do
+      post_json = {
+        animal: {
+          name: "dog",
+          body: "dope"
+        }
+      }
+
+      prev_count = Animal.count
+      post(:create, params: post_json, format: :json)
+      expect(Animal.count).to eq(prev_count + 1)
+      expect(Animal.where(name: "dog")).to exist
+    end
```

Comment:
this happens to be passing, but is not necessary, and doesnt actually test the functionality the way it is intended

- we dont actually send json to this endpoint. This endpoint is a non-api controller, that receives standard params.
- We dont need a controller test for it, the capybara test is fine for features that are not made with react
We can likely delete this